frontend/css/homepage.less

```diff
@@ -459,7 +459,7 @@
         top: 0;
       }
       .homepage-info {
-        bottom: 1.5em;
+        bottom: 4.5em;
```

Comment:
I think this should probably be reverted, on mobile phone in horizontal mode the bottom text comes butting in with the top text:
![image](https://github.com/user-attachments/assets/f074dce7-e0ce-4a45-9fb5-7e524f529272)

With the previous `1.5em` value:
![image](https://github.com/user-attachments/assets/e210060f-3660-46e7-b894-9c36e5ea4246)


Only applies to this `max-height` media query, the other one is fine

Reply:
But, in the cases where we show the BrainzPlayer on the homepage, it fits perfectly
![image](https://github.com/user-attachments/assets/4472417f-f5fd-4913-b42e-f65fc9edfc80)

Reply:
Ah yes, I forgot while testing that you can be playing music and then go to the landing page.